editor/animation_track_editor.cpp

```diff
@@ -1429,11 +1429,11 @@ void AnimationTrackEdit::_notification(int p_what) {
 			}
 		}
 
-		if (in_group) {
-			draw_line(Vector2(timeline->get_name_limit(), get_size().height), get_size(), linecolor);
-		} else {
-			draw_line(Vector2(0, get_size().height), get_size(), linecolor);
-		}
+		//if (in_group) {
```

Comment:
Why is this commented out? If not needed it should be removed.

Reply:
It's just an error. I'll fix it in 10 min